test/bench/utils/device_context.hpp

```diff
@@ -18,14 +18,14 @@
  *
  **************************************************************************/
 
-#ifndef PORTFFT_TEST_BENCH_UTILS_SYCL_UTILS_HPP
-#define PORTFFT_TEST_BENCH_UTILS_SYCL_UTILS_HPP
+#ifndef PORTFFT_TEST_BENCH_UTILS_DEVICE_CONTEXT_HPP
+#define PORTFFT_TEST_BENCH_UTILS_DEVICE_CONTEXT_HPP
 
 #include <sycl/sycl.hpp>
 
 #include <benchmark/benchmark.h>
 
-void print_device(sycl::queue queue) {
+void add_device_context(sycl::queue queue) {
```

Comment:
The original `print_device` description better describes what thsi function does. What is the motivation for changing the name?

Reply:
It's not really printing the device as it is adding context using the Gbench. At first I thought I could move a function named `print_device` to the new `sycl_utils.hpp` file but I couldn't because of the Gbench dependency.